### PR TITLE
fix: use jsdelivr CDN for ipywidgets

### LINKED ISSUE #458: ipywidgets no longer rendering
### Describe the bug

**expectation/bug**
the ipywidget should appear in the docs, but doesn't

By looking at the issues history, seems like a recurrent problem, and it is back.

Ipywidgets no longer appear in rendered html pages, and you can actually see it in the official docs :

See it appearing for 0.16
https://myst-nb.readthedocs.io/en/v0.16.0/render/interactive.html#ipywidgets

And see it not appearing for 0.17
https://myst-nb.readthedocs.io/en/v0.16.0/render/interactive.html#ipywidgets

I can confirm for my own docs that 0.17.0 does not show ipywidgets

Interestingly, widgets do appear in the next page about inline code in the official docs, see https://myst-nb.readthedocs.io/en/v0.17.0/render/inline.html#inserting-different-output-types

### Reproduce the bug

just building this repo doc will do I believe.

I also tried to build docs with 0.16 and it didn't work as well, contrary to online docs.

### List your environment

Jupyter Book      : 0.13.1
External ToC      : 0.2.4
MyST-Parser       : 0.18.1
MyST-NB           : 0.17.1
Sphinx Book Theme : 0.3.3
Jupyter-Cache     : 0.5.0
NbClient          : 0.5.13

### PR DESCRIPTION
**Pull request by agoose77**
Should fix #458, supercedes https://github.com/executablebooks/MyST-NB/pull/469

### CODE REVIEW

**Comment by agoose77** [comment]
@choldgraf let's see the rendered PR docs, and judge if this did indeed work!

File: myst_nb/core/config.py [hunk 1]
```diff
@@ -64,7 +64,7 @@ def ipywidgets_js_factory() -> Dict[str, Dict[str, str]]:
             "crossorigin": "anonymous",
         },
         # Load IPywidgets bundle for embedding.
-        "https://unpkg.com/@jupyter-widgets/html-manager@^0.20.0/dist/embed-amd.js": {
+        "https://cdn.jsdelivr.net/npm/@jupyter-widgets/html-manager@1.0.6/dist/embed-amd.js": {
```
Comment (choldgraf) [inline myst_nb/core/config.py:67]: can we add a comment here that provides a loose guidelines for when/how to update this import link? Or maybe just a block comment at the top of the `#Load IPywidgets..` section?

**Review by choldgraf — DISMISSED** [review]
I believe that it works! Good catch:

- Master: https://myst-nb.readthedocs.io/en/stable/render/interactive.html#ipywidgets
- This PR: https://myst-nb--491.org.readthedocs.build/en/491/render/interactive.html#ipywidgets

File: myst_nb/core/config.py (hunk 1 above)
Comment (agoose77) [reply myst_nb/core/config.py:67]: Hehe, we're on the same wavelength: https://github.com/executablebooks/MyST-NB/pull/491/commits/2deaa406b91f1f6f7e93db27ef9c9befc41fb053

**Comment by agoose77** [comment]
@choldgraf merge if you're happy!

**Review by choldgraf — APPROVED** [review]
Looks great, thanks!

**Pull request merged**